Dockerfile

```diff
@@ -0,0 +1,14 @@
+# syntax=docker/dockerfile:1
+
+FROM python:3.6-slim-buster
+
+WORKDIR /app
+
+COPY requirements/requirements.txt requirements.txt
+RUN pip3 install -r requirements.txt
+
+COPY . .
+
+WORKDIR /app/src
+
+CMD [ "python3", "-m" , "flask", "run", "--host=0.0.0.0"]
```

Comment:
Ah, this does not make much sense anymore since there is no rest api running

Reply:
You are right. I'll delete it